docs/BuildConfiguration.md

```diff
@@ -12,13 +12,16 @@ Sections follow that describe particular settings in more depth.
 | `blockBasedInterlining`            | Whether to automatically insert an in-seat transfer between two trips with the same block id.                                                                                                                                                                                    | boolean        | true                    | Consider using using [transfer types 4 and 5](https://github.com/google/transit/pull/303) |
 | `boardingLocationTags`             | The OSM tags to extract a stop's boarding location from.                                                                                                                                                                                                                         | list           | `["ref"]`               | [Detailed documentation](./BoardingLocations.md)                                          |
 | `dataImportReport`                 | Generate nice HTML report of Graph errors/warnings                                                                                                                                                                                                                               | boolean        | false                   |                                                                                           |
+| `dem`                              | Configure access to elevation data.                                                                                                                                                                                                                                              | array          | null                    |                                                                                           |
```

Comment:
Maybe we should link the appropriate chapter in this file from these new descriptions as otherwise it can be confusing to understand what type of an array you need to provide.